From adacb2d6701b12511c5a0e58072236da91ebf810 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Mon, 7 Aug 2017 22:54:11 +0200
Subject: [PATCH] examples: fix compile without opengl support

Compile examples/opengl only in case opengl support is available.

Task-number: QTBUG-62372
Change-Id: I742a1eb7b7639a5a722c4d5e9b4ee070b629b02e

Upstream: https://codereview.qt-project.org/201947
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 examples/examples.pro | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/examples/examples.pro b/examples/examples.pro
index a3851c6d81..d87fa2da88 100644
--- a/examples/examples.pro
+++ b/examples/examples.pro
@@ -14,7 +14,8 @@ qtHaveModule(concurrent): SUBDIRS += qtconcurrent
 qtHaveModule(sql): SUBDIRS += sql
 qtHaveModule(widgets): SUBDIRS += widgets
 qtHaveModule(xml): SUBDIRS += xml
-qtHaveModule(gui): SUBDIRS += gui opengl
+qtHaveModule(gui): SUBDIRS += gui
+qtHaveModule(gui):qtConfig(opengl): SUBDIRS += opengl
 
 aggregate.files = aggregate/examples.pro
 aggregate.path = $$[QT_INSTALL_EXAMPLES]
-- 
2.11.0

